daemon/integration/python/integration_py_test.go

```diff
@@ -1,91 +1,39 @@
 package integration_py_test
 
 import (
-	"flag"
 	"os"
 	"os/exec"
-	"path"
 	"testing"
-	"time"
 
 	"github.com/scootdev/scoot/daemon/server"
-	"github.com/scootdev/scoot/os/temp"
-	"github.com/scootdev/scoot/runner"
-	os_execers "github.com/scootdev/scoot/runner/execer/os"
-	"github.com/scootdev/scoot/runner/local"
-	"github.com/scootdev/scoot/snapshot"
-	"github.com/scootdev/scoot/snapshot/snapshots"
 )
 
 var s *server.Server
 
 func TestMain(m *testing.M) {
-	flag.Parse()
-	tempDir, err := temp.NewTempDir("", "scoot-listen-")
-	if err != nil {
-		panic(err)
-	}
-	defer os.RemoveAll(tempDir.Dir)
-
-	scootDir := path.Join(tempDir.Dir, "scoot")
-	err = os.Setenv("SCOOTDIR", scootDir)
-
-	filer := snapshots.MakeTempFiler(tempDir)
-	h := server.NewHandler(getRunner(filer), filer, 50*time.Millisecond)
-	s, err := server.NewServer(h)
-	if err != nil {
-		panic(err)
-	}
-
-	l, _ := s.Listen()
-	go func() {
-		s.Serve(l)
-	}()
-
-	defer s.Stop()
-
 	os.Exit(m.Run())
 }
 
 func TestPython(t *testing.T) {
+	// each test starts and stops the daemon, run the tests one at a time
```

Comment:
don't want the test platform to run them in parallel, so putting them in one test